script/draft-release/channel.ts

```diff
@@ -0,0 +1 @@
+export type Channel = 'production' | 'beta'
```

Comment:
Should we add `test` and `development` as well since `app.tsx` uses the `__RELEASE_CHANNEL__ constant?

https://github.com/desktop/desktop/blob/484856d67a7975aa4d574b7966355a7b873fff92/app/src/lib/globals.d.ts#L27-L31

Reply:
We should only focus on the channels that we actually publish too, so I wouldn't worry about `development` here. I'll add `test` to this list but I won't be implementing it as part of this PR.